source/content/guides/support/02-contact-support.md

```diff
@@ -71,12 +69,12 @@ You can download a transcript of your chat with the <Icon icon="arrowDownToLine"
 
 <Alert title="Note" type="info">
 
-Some in-browser ad or tracking blockers can interfere with the **Launch Chat** button. If you don't see it, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has posted more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
+Some in-browser ad or tracking blockers may interfere with the **Start chat** button. If you don't see the button, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has provided more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
 
 </Alert>
 
-## Premium Technical Support and On-Call Services
+## Premium Support Hotline and Emergency Ticket
 
-Diamond and Platinum Account customers can call Pantheon's premium technical support line directly for any technical issues, escalations, site, billing, or overages queries. The phone number can be found in your Workspace, in the Support tab.
+**Diamond** and **Platinum** customers can call the 24/7 Premium Support Hotline for urgent account-related issues. You can find the phone number in the Support tab of your workspace. The phone number can be found in the Support tab of your workspace.
```

Comment:
Second location where we're mentioning the scope of Phone support. I can't verify on my end that this is a real requirement change, and haven't been informed of this, if it's happening.

Reply:
Same comment I listed above - we are changing copy on the Support page in the new dashboard experience and are not changing the scope or type of support being offered. If this is confusing I'd recommend leaving as is and then revisiting in the near future